Fixes #395 add new shapes

pchelle opened the pull request:
Examples were added to the documentation, which provides a visual display of the linetypes and shapes in the website. For this reason, the doc website was also updated.

Also, note that this fixes a ggplot2 issue that prevents using some shapes together.

PavelBal commented:
@pchelle Are these symbols actually supported? https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/413

pchelle commented:
> @pchelle Are these symbols actually supported? #413

Yes. They are supported, the issue is that they cannot be combined with legal ggplot2 shapes ("circle", "square", etc) because they do not share the same format: https://stackoverflow.com/questions/67859447/error-when-combining-unicode-shape-with-legal-ggplot2-shapes

This is exactly the reason why I chose to include only utf characters for drawing all the symbols available in `Shapes`.
I referenced the issue within the documentation of `Shapes` so that it is also available in the website.

Yuri05 commented:
@pchelle Why are the symbols in the new plots so much smaller than before?
![grafik](https://user-images.githubusercontent.com/25061876/222454067-8f0435dd-2e55-437a-8af8-f8d31675f6af.png)

Yuri05 commented:
same here:
https://github.com/Open-Systems-Pharmacology/TLF-Library/pull/414/files?file-filters%5B%5D=.R&file-filters%5B%5D=.json&file-filters%5B%5D=.png&file-filters%5B%5D=.xml&file-filters%5B%5D=.yml&show-viewed-files=false#diff-75802937aa1e49a25f3ec6b5f81f672a834536e8530504d8feba9267ac7717d2
![grafik](https://user-images.githubusercontent.com/25061876/222455115-2f11df75-8e8b-4384-8e9e-38007db7884e.png)

Yuri05 commented:
>I am updating circle and square, this should be better but might still be slightly smaller than ggplot initial symbols.

@pchelle Can you please regenerate the docu as well?

Yuri05 commented:
@pchelle Smybols are bigger now, but I wonder why e.g. squares became circles now?
![grafik](https://user-images.githubusercontent.com/25061876/222491873-0309a878-a1b7-44a8-97e5-f5d68aef9c47.png)

pchelle commented:
> @pchelle Smybols are bigger now, but I wonder why e.g. squares became circles now? ![grafik](https://user-images.githubusercontent.com/25061876/222491873-0309a878-a1b7-44a8-97e5-f5d68aef9c47.png)

I have changed the order of symbols in the themes.
Before the shapes were square, **dot**, circle etc but since dot was not useful I changed it to circle, square, diamond, triangle, etc.
That way the default first shape used in most of the plots is a circle. But I can revert to square if necessary.

Yuri05 commented:
no need to revert; I just wanted to be sure that we don't have a square in the enum represented by circle or so :)

Yuri05 commented:
not yet ready for review

pchelle commented on docs/dev/reference/Shapes-1.png:
This plot (included in the website documentation) displays the current available Shapes

pchelle commented on docs/dev/reference/Linetypes-1.png:
This plot (included in the website documentation) displays the current available Linetypes
